### DISCUSSION TITLE
in-network files with the same url but for a different month

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#658] [Q&A]
The specification for the file naming convention says that there is no naming convention for the in-network files when they are used as part of a table-of-contents file:

> If multiple plans are to be included in a single file, a table-of-contents file will be required. The naming standard will be applied to the table-of-contents file and both the in-network and allowed-amounts files will not have any naming standards.

This introduce the possibility of the in-network for two separate months having the same file name (and url). This is rather inconvenient since it is harder to determine if the file has been changed from one check to the next.  Some payers are doing this: simply updating the same file from month to month.

I would suggest that it would be much clearer if the in-network files were required to include a date in the file name even if they are only referenced by a table of contents file.  The inconvenience to the payers of this requirement would be extremely small and the benefit to the users would be great.

Thoughts?

**Comment by dgolden** [comment]
I agree; including the date in the filename is a best practice across a range of industries.

I would go further, and mandate a standard naming convention for MRF files, whether they are published stand-alone or are referenced via table-of-contents file.

**Reply by bryanbriegel** [reply]
While the naming convention for a table of contents file includes date: table-of-contents file: YYYY-MM-DD _ payer or issuer name _ index. file extension - as you note, with no standard naming convention for the files being referenced, there are issues.  Ex today we were assessing a February-dated Table of Contents file, and the URLs in the file contained January.  So we see Feb ToC and Jan files -only as the producer elected to label the rate files with a date (as noted it is not required) from a public consumption standpoint,   requiring a date in all file names is needed.
